Servers/UI/OJS.Servers.Ui/ClientApp/src/common/url-types.ts

```diff
@@ -51,6 +51,11 @@ interface IGetSubmissionsUrlParams {
     page: number;
 }
 
+interface IGetSubmissionsByContestIdParams {
+    page: number;
+    contestId: string | string[] | undefined;
```

Comment:
Contest id should not be string[]

Isnt undefined default value either way?

Reply:
It is, bit since I get it via .find of the url params, for some reason typescripts sets the return value of the function to string | string[] | undefined, not sure why is that but it doesnt seem to allow me to to map it to this interface if I remove the string[]. However the return value of .find is never an array, not sure why is this TS behaviours. The issue arrises in initiateSubmissionsByContestQuery function of use-public-submissions.tsx file where I am getting the value of contestIdParam